packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md

```diff
@@ -8,7 +8,24 @@ title: Download and set up the SDK, Sign-In Widget, and sample app
 
 <StackSelector class="cleaner-selector"/>
 
-After you [create and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/), the next step is to download the GitHub repository that contains the SDK, Widget, and accompanying sample apps.
+This guide covers the next step after you [create and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/aspnet/main/), which is to download the GitHub repository that contains the SDK, Widget, and accompanying sample apps.
+
+---
+**Learning outcomes**
+* Configure the settings that you need to add to initialize the SDK and the Sign-In Widget.
+* Integrate the SDK and the Sign-In Widget into your own app after you run the sample app.
+
+**What you need**
+[Software Requirements](/docs/guides/oie-embedded-common-download-setup-app/android/main/#software-requirements).
```

Comment:
I think in general you need to add a blank line in between each bolded "heading" line and the nutrition facts content that appears beneath each one. When I look at the rendered version of this, the "Software Requirements" link appears on the same line as "What you need".

Reply:
Also "Software Requirements" should be "Software requirements"

Reply:
Fixed.